staging/src/kubevirt.io/client-go/kubecli/replicaset.go

```diff
@@ -87,8 +87,8 @@ func (v *rc) List(options k8smetav1.ListOptions) (replicasetList *v1.VirtualMach
 	return
 }
 
-func (v *rc) Create(replicaset *v1.VirtualMachineInstanceReplicaSet) (result *v1.VirtualMachineInstanceReplicaSet, err error) {
-	result, err = v.VirtualMachineInstanceReplicaSetInterface.Create(context.Background(), replicaset, k8smetav1.CreateOptions{})
+func (v *rc) Create(ctx context.Context, replicaset *v1.VirtualMachineInstanceReplicaSet, opts k8smetav1.CreateOptions) (result *v1.VirtualMachineInstanceReplicaSet, err error) {
```

Comment:
Please consider dropping the named return values.
Same for other methods.

Reply:
I think it is fine as is for the moment.
We can think about defining a generalized behavior for all the methods in a follow up PR.
WDYT?

Reply:
I cannot see the misalignment you are referring to...
All the methods use the named returning values except `Delete` (that returns directly the generated function output).

Reply:
It could also be seen under the `ReplicaSetInterface`.

Reply:
Consider that in https://github.com/kubevirt/kubevirt/pull/11685 (or similar) interface will be moved in the `vmrs_expansion`. We can make it standard there :) WDYT?